### DISCUSSION TITLE
AMBITIOS IDEAS/PROJECT

### DISCUSSION THREAD

**Discussion by FelipePenhaa** [elementary/triage#532] [Ideas]
hi, i have huge ambitious ideas that i am giving to all linux distributions:
obs: Linux itself can still be open source

-unify/merge all distro companies/organizations while also merging with linux organization to work with each other(remotely if necessary) towards a single linux distribution/os with all the features of every distribution while forsaking the current ones

-make deals with softwares and computer/notebooks companies in order for linux to come installed as you buy a new computer/notebook and to support more softwares 

-improve marketing by for example sponsoring content creators and financing ads throughout the internet, the linux organization is actually one of the main culprits here for the low fame of linux, they alledlgy spend only 3% of all their profits on the development of the linux kernel while spending all the reamaining money on a bunch of useless things which have nothing to do with marketing at all(i believe someone/some company related to linux should actually sue them and report them to police for not spending their money properly on what they claim and they should. to me it just sounds like some sort of corruption scheme to divert the money to themselves personally in the end, like equality and racially profilling ''projects/activism'' that having nothing to do at all with the development of linux)

-create a pro and free version and charge on a monthly basis for the pro(since employees are paid monthly wages, the software is continuously developed and considering it can pay off more than a lifetime license) 
-still accept donations

the benefits will be: 
obs: turning into a corporate business model would allow the project to be more professional and increase its development organization, it does not mean however that the public would not be allowed to help its development, they still could and even get paid for it as an incentive.

-more developers working in a single distribution allowing for a better distribution with faster improvements and bug fixes  

-tighter control over the software

-faster overall development of the os

-higher profits which can in the end be used to boost the development of the os and pay its emplyees/devs better.

some main features a os should have natively:
driver updater and softwares updater/backports repo
cloud backup
great user interface
notifications
notification panel
complete customizations> such as> cursor size / font size/ font type/ windows icons size/ taskbar location/ taskbar size
overclock tools
antivirus
settings
desktop
desktop folders
file manager
taskbar
task manager
startmenu
complete clipboard manager
password manager
notes
media viewer and player
resource monitor 
apps manager/viewer
native feedback app(for bug report and features request)

**Comment by RMcNeely** [comment]
Some of these things are done already. For instance there are OEMs that ship elementary and other Linux distros like Mint or Ubuntu that elementary does receive a (small) cut of. Star Labs Slimbook and Laptops with Linux are some retailers that support elementary out of the box. You can find out more on the [OEM](https://elementary.io/oem) page of the website and on the [store](https://store.elementary.io). 

Mostly what you're talking about though is essentially RHEL (and to a similar extent Ubuntu) which already exists in that the bigger companies do sell support to go along with the open source offering. All distros just make their own spin on what they find is unique and important and you'll never get everyone to go along together on  "the one true Linux distro".  This approach lets folks chose what they want while cross distro efforts like Flathub help solve the problems of high quality apps for everyone no matter what base they come from.
